lib/Limitation/Templating/LimitationBlockRenderer.php

```diff
@@ -70,9 +70,13 @@ public function renderLimitationValue(Limitation $limitation, array $parameters
         return $template->renderBlock($blockName, $parameters);
     }
 
-    public function setLimitationValueResources(array $limitationValueResources)
+    public function setLimitationValueResources(array $resources)
     {
-        $this->limitationValueResources = $limitationValueResources;
+        usort($resources, function ($a, $b) {
+            return $b['priority'] - $a['priority'];
```

Comment:
Minor thing but since we encourage to use PHP 7 you could have used spaceship operator here (<=>).

Reply:
> Minor thing but since we encourage to use PHP 7 you could have used spaceship operator here (<=>).

Noooo... :) It's `1.x`, so should work with PHP 5.6